apps/chat/src/backend/serverless/template.yaml

```diff
@@ -190,6 +302,28 @@ Resources:
               - "x-amz-id-2"
             MaxAge: "3000"
 
+  # Creates a role that allows Cognito to send SNS messages

```

Comment:
This role is not used anywhere. Can be removed

Reply:
updated